src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.Registry/Commands/CopyValueNameCommand.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Resources;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CmdPal.Ext.Registry.Classes;
+using Microsoft.CmdPal.Ext.Registry.Helpers;
+using Microsoft.CmdPal.Ext.Registry.Properties;
+using Microsoft.CmdPal.Extensions;
+using Microsoft.CmdPal.Extensions.Helpers;
+using Windows.ApplicationModel.DataTransfer;
+using Windows.UI;
+
+namespace Microsoft.CmdPal.Ext.Registry.Commands;
+
+internal sealed partial class CopyValueNameCommand : InvokableCommand
+{
+    private readonly RegistryEntry _entry;
+
+    internal CopyValueNameCommand(RegistryEntry entry)
```

Comment:
you could _probably_ dedupe this with CopyValueDataCommand by just passing in a `bool copyName` to indicate if you want to copy the name or data

Reply:
actually the same with the CopyKey one above that too. Maybe pass in an enum for which of the three you want to copy